charts/application/values.yaml

```diff
@@ -112,6 +112,11 @@ resources: {}
 #    cpu: 2
 #    memory: 3000Mi
 
+# The lifecycle handlers for the container
+lifecycle:
```

Comment:
can you please comment the k8s docu link with the structure of postStart and preStop